torchtune/__init__.py

```diff
@@ -23,6 +23,18 @@
         """
     ) from e
 
+# Enables faster downloading. For more info: https://huggingface.co/docs/huggingface_hub/en/guides/download#faster-downloads
+# To disable, run `HF_HUB_ENABLE_HF_TRANSFER=0 tune download <model_config>`
+try:
+    import os
+
+    import hf_transfer  # noqa
+
+    if os.environ.get("HF_HUB_ENABLE_HF_TRANSFER") is None:
```

Comment:
does this break if the environ variable is not present?
```suggestion
    if os.environ.get("HF_HUB_ENABLE_HF_TRANSFER", None) is None:
```